dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -570,98 +674,108 @@ public void run(){
+                            endpoint.getId(),
+                            url,
+                            MediaType.APPLICATION_JSON_TYPE,
+                            Entity.entity(form, form.getMediaType()));
 
-                    if ( response.getStatus() == HttpStatus.SC_OK ) {
+                    if (response.getStatus() == HttpStatus.SC_OK) {
                         //Nothing to do here, we found no process to cancel
-                    } else if ( response.getStatus() == HttpStatus.SC_RESET_CONTENT ) {
+                    } else if (response.getStatus() == HttpStatus.SC_RESET_CONTENT) {
                         //Expected return status if a cancel was made on the end point server
                     } else {
-                        Logger.error( this.getClass(), "Response indicating a " + response.getStatusInfo().getReasonPhrase() + " (" + response.getStatus() + ") Error trying to interrupt the running process on the Endpoint [ " + endpointId + "]." );
+                        Logger.error(
+                                this.getClass(),
+                                "Response indicating a " + response.getStatusInfo().getReasonPhrase() + " ("
+                                        + response.getStatus()
+                                        + ") Error trying to interrupt the running process on the Endpoint [ "
+                                        + endpointId + "].");
                     }
 
                     //Interrupt the Thread process
                     runningThread.interrupt();
-
                     //Remove the thread from the session
-                    clearThreadInSession( httpServletRequest, endpointId );
+                    clearThreadInSession(httpServletRequest, endpointId);
 
-                    jsonResponse.put( "success", true );
-                    jsonResponse.put( "message", LanguageUtil.get( initData.getUser().getLocale(), "IntegrityCheckingCanceled" ) );
-                } else {
-                    jsonResponse.put( "success", false );
-                    jsonResponse.put( "message", "The integrity process for End Point server: [" + endpointId + "] was already stopped." );
+                    jsonResponse.put("success", true);
+                    jsonResponse.put(
+                            "message",
+                            LanguageUtil.get(initData.getUser().getLocale(), "IntegrityCheckingCanceled"));
                 }
             }
 
-            responseMessage.append( jsonResponse.toString() );
-
-        } catch ( Exception e ) {
-            Logger.error( this.getClass(), "Error checking the integrity process status for End Point server: [" + endpointId + "]", e );
+            responseMessage.append(jsonResponse.toString());
+        } catch (Exception e) {
+            Logger.error(
+                    this.getClass(),
+                    "Error checking the integrity process status for End Point server: [" + endpointId + "]",
```

Comment:
Codacy found an issue: [The String literal "Error checking the integrity process status for End Point server: [" appears 5 times in this file; the first occurrence is on line 667](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)